private/bufpkg/bufreflect/bufreflect_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bufreflect_test
+
+import (
+	"testing"
+
+	"github.com/bufbuild/buf/private/bufpkg/bufreflect"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseFullyQualifiedModuleTypeName(t *testing.T) {
```

Comment:
This test name should be updated to match the function name.
```suggestion
func TestParseFullyQualifiedPath(t *testing.T) {
```